### PR TITLE
style(myprofile): Bootstrap adaptation

### PR DESCRIPTION
**Pull request by xxxrenegade**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL/commit/7226839f2397883a39f535fe930145f2d1ce5d4d%23commitcomment-11177372%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-105289780%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-106242804%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-109557933%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-109721952%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-114142491%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23discussion_r33125499%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-114791930%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-123387354%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-123390680%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-127051120%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23issuecomment-105289780%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Passt%20der%20Pull%20requests%3F%20Fehlt%20nocht%20was%3F%20Soll%20was%20ge%5Cu00e4ndert%20werden%3F%22%2C%20%22created_at%22%3A%20%222015-05-25T19%3A04%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22Mach%20bitte%20alle%20whitespace%20%5Cu00c4nderungen%20r%5Cu00fcckg%5Cu00e4ngig%20-%20dann%20squash%20deine%20commits%20in%20einen.%5Cr%5CnOb%20die%20spaces%20vor%20und%20nach%20eckigen%20klamern%20richtig%20sind%20oder%20nicht%20-%20diese%20%5Cu00c4nderung%20sollte%20einmal%20f%5Cu00fcr%20alle%20Dateien%20gemacht%20werden%20-%20das%20ist%20auserhalb%20des%20scopes%20dieses%20PR.%22%2C%20%22created_at%22%3A%20%222015-05-28T09%3A09%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22Wie%20soll%20man%20den%20jetzt%20mit%20dem%20hinzuf%5Cu00fcgen%20von%20neuen%20Sprachvariablen%20umgehen%3F%20Wegen%20Crowdin%20meine%20ich.%20Einfach%20immer%20nur%20in%20Englisch%20hinzuf%5Cu00fcgen%3F%20Habe%20gerade%20gesehen%20das%20%27e-mail%27%20auf%20Crowdin%20schon%20mit%20drin%20ist.%20Hier%20auf%20Github%20allerdings%20nicht%2C%20nur%20in%20uk%20aber%20nicht%20in%20en.%22%2C%20%22created_at%22%3A%20%222015-06-06T09%3A34%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22Neue%20Sprachvariablen%20einfach%20nur%20auf%20Crowdin%20am%20besten%20hinzuf%5Cu00fcgen%20sonst%20gibt%20es%20Probleme%20wenn%20wir%20die%20Sprachen%20r%5Cu00fcber%20holen%22%2C%20%22created_at%22%3A%20%222015-06-07T08%3A34%3A32Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22Passt%20der%20Pull%20requests%3F%20Fehlt%20nocht%20was%3F%20Soll%20was%20ge%5Cu00e4ndert%20werden%3F%22%2C%20%22created_at%22%3A%20%222015-06-22T15%3A03%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20know%20this%20has%20been%20wrong%20before%2C%20but%20it%20is%20still%20wrong%20now.%20%60col-%2A%2A-%2A%2A%60%20has%20to%20be%20a%20child%20of%20a%20%60row%60%5Cr%5Cn%5Cr%5Cn%3E%20Content%20should%20be%20placed%20within%20columns%2C%20and%20only%20columns%20may%20be%20immediate%20children%20of%20rows.%5Cr%5Cn%5Cr%5Cn%28de%29%20http%3A//holdirbootstrap.de/css/%23grid-intro%5Cr%5Cn%28en%29%20http%3A//getbootstrap.com/css/%23grid-intro%22%2C%20%22created_at%22%3A%20%222015-06-24T08%3A51%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%21%5Bimage%5D%28https%3A//cloud.githubusercontent.com/assets/498197/8805832/a97d274a-2fd3-11e5-8d4c-5a750db9376c.jpg%29%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222015-07-21T16%3A09%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ich%20denke%20das%20dieser%20PR%20richtige%20%5Cu00c4nderungen%20enth%5Cu00e4lt%2C%20es%20sind%20noch%20immer%20Whitespace%20%5Cu00c4nderungen%20vorhanden%20wo%20mir%20die%20Motivation%20unklar%20ist.%20Das%20Diff%20ist%20dadurch%20unn%5Cu00f6tig%20gro%5Cu00df%20%28unheimlich%20schwierig%20zu%20erkennen%20was%20ge%5Cu00e4ndert%20wurde%29.%20Das%20Testen%20ob%20deine%20Anpassung%20responsive%20ist%20h%5Cu00e4ttest%20du%20schon%20vorher%20machen%20k%5Cu00f6nnen...%5Cr%5Cn%5Cr%5CnEin%20%60git%20rebase%60%20solltest%20du%20zum%20schluss%20noch%20machen%2C%20es%20sind%20ja%20mittlerweile%20%5Cu00fcber%2020%20commits%2C%20einer%20w%5Cu00fcrde%20hier%20wohl%20auch%20reichen.%22%2C%20%22created_at%22%3A%20%222015-07-21T16%3A20%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22Der%20Pull%20kann%20geschlossen%20werden%2C%20ich%20habe%20gerade%20einen%20neuen%20gestellt.%20Ich%20denke%20dort%20ist%20alles%20etwas%20ordentlicher.%20Au%5Cu00dferdem%20habe%20ich%20meine%20Arbeitsumgebung%20neu%20aufgesetzt%20und%20auch%20meinen%20webSPELL%20Fork%20neu%20angelegt.%22%2C%20%22created_at%22%3A%20%222015-08-02T18%3A04%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Commit%207226839f2397883a39f535fe930145f2d1ce5d4d%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/commit/7226839f2397883a39f535fe930145f2d1ce5d4d%23commitcomment-11177372%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22The%20majority%20of%20the%20changes%20are%20codestyle%20related.%5Cr%5CnNot%20sure%20where%20they%20are%20coming%20from%2C%20like%20SQL%20re-formatted.%22%2C%20%22created_at%22%3A%20%222015-05-13T14%3A43%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22Commit%207226839%20comments%22%7D%2C%20%22Pull%20120e2362516642a651de830839acc3a1e2cb8c16%20myprofile.php%2048%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/244%23discussion_r33125499%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20did%20you%20change%20this%3F%20Revert%20please.%22%2C%20%22created_at%22%3A%20%222015-06-24T08%3A04%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20myprofile.php%3AL150-178%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Commit 7226839f2397883a39f535fe930145f2d1ce5d4d'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/commit/7226839f2397883a39f535fe930145f2d1ce5d4d#commitcomment-11177372'>Commit 7226839 comments</a></b>
- <a href='https://github.com/derchrisuk'></a> The majority of the changes are codestyle related.
Not sure where they are coming from, like SQL re-formatted.
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/244#issuecomment-105289780'>General Comment</a></b>
- <a href='https://github.com/xxxrenegade'></a> Passt der Pull requests? Fehlt nocht was? Soll was geändert werden?
- <a href='https://github.com/Pascalmh'></a> Mach bitte alle whitespace Änderungen rückgängig - dann squash deine commits in einen.
Ob die spaces vor und nach eckigen klamern richtig sind oder nicht - diese Änderung sollte einmal für alle Dateien gemacht werden - das ist auserhalb des scopes dieses PR.
- <a href='https://github.com/xxxrenegade'></a> Wie soll man den jetzt mit dem hinzufügen von neuen Sprachvariablen umgehen? Wegen Crowdin meine ich. Einfach immer nur in Englisch hinzufügen? Habe gerade gesehen das 'e-mail' auf Crowdin schon mit drin ist. Hier auf Github allerdings nicht, nur in uk aber nicht in en.
- <a href='https://github.com/Phhere'></a> Neue Sprachvariablen einfach nur auf Crowdin am besten hinzufügen sonst gibt es Probleme wenn wir die Sprachen rüber holen
- <a href='https://github.com/xxxrenegade'></a> Passt der Pull requests? Fehlt nocht was? Soll was geändert werden?
- <a href='https://github.com/Pascalmh'></a> I know this has been wrong before, but it is still wrong now. `col-**-**` has to be a child of a `row`
> Content should be placed within columns, and only columns may be immediate children of rows.
(de) http://holdirbootstrap.de/css/#grid-intro
(en) http://getbootstrap.com/css/#grid-intro
- <a href='https://github.com/Pascalmh'></a> image
- <a href='https://github.com/Pascalmh'></a> Ich denke das dieser PR richtige Änderungen enthält, es sind noch immer Whitespace Änderungen vorhanden wo mir die Motivation unklar ist. Das Diff ist dadurch unnötig groß (unheimlich schwierig zu erkennen was geändert wurde). Das Testen ob deine Anpassung responsive ist hättest du schon vorher machen können...
Ein `git rebase` solltest du zum schluss noch machen, es sind ja mittlerweile über 20 commits, einer würde hier wohl auch reichen.
- <a href='https://github.com/xxxrenegade'></a> Der Pull kann geschlossen werden, ich habe gerade einen neuen gestellt. Ich denke dort ist alles etwas ordentlicher. Außerdem habe ich meine Arbeitsumgebung neu aufgesetzt und auch meinen webSPELL Fork neu angelegt.
- [ ] <a href='#crh-comment-Pull 120e2362516642a651de830839acc3a1e2cb8c16 myprofile.php 48'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/244#discussion_r33125499'>File: myprofile.php:L150-178</a></b>
- <a href='https://github.com/Pascalmh'></a> Why did you change this? Revert please.

<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/244?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/244?approve=1'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL/pull/244'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by webSPELL-org** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL/pull/244'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by xxxrenegade** [comment]
Passt der Pull requests? Fehlt nocht was? Soll was geändert werden?

**Comment by Pascalmh** [comment]
Mach bitte alle whitespace Änderungen rückgängig - dann squash deine commits in einen.
Ob die spaces vor und nach eckigen klamern richtig sind oder nicht - diese Änderung sollte einmal für alle Dateien gemacht werden - das ist auserhalb des scopes dieses PR.

**Comment by xxxrenegade** [comment]
Wie soll man den jetzt mit dem hinzufügen von neuen Sprachvariablen umgehen? Wegen Crowdin meine ich. Einfach immer nur in Englisch hinzufügen? Habe gerade gesehen das 'e-mail' auf Crowdin schon mit drin ist. Hier auf Github allerdings nicht, nur in uk aber nicht in en.

**Comment by Phhere** [comment]
Neue Sprachvariablen einfach nur auf Crowdin am besten hinzufügen sonst gibt es Probleme wenn wir die Sprachen rüber holen

**Comment by xxxrenegade** [comment]
Passt der Pull requests? Fehlt nocht was? Soll was geändert werden?

File: myprofile.php [hunk 1]
```diff
@@ -148,33 +150,28 @@
...
+                            $file = $id . $endung;
+                            if ($upload->saveAs($filepath . $file, true)) {
+                                @chmod($filepath . $file, $new_chmod);
+                                safe_query("UPDATE " . PREFIX . "user
                                     SET
-                                        avatar='" . $file .
-                                    "' WHERE
-                                        userID='" . $id . "'"
-                                );
+                                        avatar='" . $file . "' WHERE
```
Comment (Pascalmh) [inline myprofile.php]: Why did you change this? Revert please.

**Comment by Pascalmh** [comment]
I know this has been wrong before, but it is still wrong now. `col-**-**` has to be a child of a `row`

> Content should be placed within columns, and only columns may be immediate children of rows.

(de) http://holdirbootstrap.de/css/#grid-intro
(en) http://getbootstrap.com/css/#grid-intro

**Comment by Pascalmh** [comment]
image

**Comment by Pascalmh** [comment]
Ich denke das dieser PR richtige Änderungen enthält, es sind noch immer Whitespace Änderungen vorhanden wo mir die Motivation unklar ist. Das Diff ist dadurch unnötig groß (unheimlich schwierig zu erkennen was geändert wurde). Das Testen ob deine Anpassung responsive ist hättest du schon vorher machen können...

Ein `git rebase` solltest du zum schluss noch machen, es sind ja mittlerweile über 20 commits, einer würde hier wohl auch reichen.

**Comment by xxxrenegade** [comment]
Der Pull kann geschlossen werden, ich habe gerade einen neuen gestellt. Ich denke dort ist alles etwas ordentlicher. Außerdem habe ich meine Arbeitsumgebung neu aufgesetzt und auch meinen webSPELL Fork neu angelegt.
